javascript/js-array-methods/js-array-methods_map/index.js

```diff
@@ -24,10 +24,47 @@ const cards = [
   },
 ];
 
-const lowerCaseAnswers = null; // ['as often as you like.', ...]
+// const lowerCaseAnswers = null; // ['as often as you like.', ...]
+const lowerCaseAnswers = cards.map((card) => {
+  return card.answer.toLowerCase();
+});
+console.log(lowerCaseAnswers);
 
-const questionsAndAnswersTogether = null; // ["How often can I use <header>? - As often as you like.", ...]
+// const questionsAndAnswersTogether = null; // ["How often can I use <header>? - As often as you like.", ...]
+const questionsAndAnswersTogether = cards.map((card) => {
+  return `${card.question} - ${card.answer}`;
+});
+console.log(questionsAndAnswersTogether);
 
-const questionAndAnswer = null; // [{ question: 'How often can I use <header>?', answer: 'As often as you like.'}, {...}]
+// const questionAndAnswer = null; // [{ question: 'How often can I use <header>?', answer: 'As often as you like.'}, {...}]
 
+const questionAndAnswer = cards.map((card) => {
+  const newObject = {
+    question: card.question,
+    answer: card.answer,
+  };
+  return newObject;
```

Comment:
Although this is totally fine and very explicit, since it's quite clear we are returning a new object anyway, you could simply do:
```suggestion
const questionAndAnswer = cards.map(card => { question: card.question, answer: card.answer };
```